packages/openneuro-app/src/scripts/users/user-query.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import * as Sentry from '@sentry/react'
+import PropTypes from 'prop-types'
+import { useParams } from 'react-router-dom'
+import { Loading } from '@openneuro/components/loading'
+import UserRoutes from './user-routes'
```

Comment:
```suggestion
import { UserRoutes } from './user-routes'
```

Reply:
I've updated files I created to be named exports. Should I go ahead and do the same for ones I see like <FourOFour /> etc.